src/wallet/wallet.h

```diff
@@ -47,9 +47,9 @@ static const unsigned int DEFAULT_KEYPOOL_SIZE = 100;
 //! -paytxfee default
 static const CAmount DEFAULT_TRANSACTION_FEE = 0;
 //! -fallbackfee default
-static const CAmount DEFAULT_FALLBACK_FEE = COIN * 20;
+static const CAmount DEFAULT_FALLBACK_FEE = COIN;
 //! -mintxfee default
-static const CAmount DEFAULT_TRANSACTION_MINFEE = COIN;
+static const CAmount DEFAULT_TRANSACTION_MINFEE = COIN / 10;
```

Comment:
Why ist this `COIN / 10`? From 1.10: `CFeeRate minRelayTxFee = CFeeRate(COIN);
`

Reply:
The idea is to intentionally loosen the minimum so we've got some scope for the price to continue upwards without causing problems.

Reply:
Given we are always creating transactions at `minRelayTxFee` this change would be effectively reducing the fee for every Core user to 0.1 DOGE. I think you meant to change `DEFAULT_MIN_RELAY_TX_FEE`?